core/src/main/scala/org/bykn/bosatsu/MainModule.scala

```diff
@@ -1146,6 +1149,21 @@ abstract class MainModule[IO[_]](implicit
       }
     }
 
+    case class Show(
+        inputs: Inputs.Runtime,
```

Comment:
it would be nice to support showing interfaces as well, but there is no defined Inputs that takes interfaces and packages only, so we would need to define a new input.